Bottom: 02d10a4165c1c6a28743800b64f1186a5a7b7391
Top:    2fdbc967fc77d22fe9ee800dc6fdf95b60e4b870
Author: Pavel Roskin <proski@gnu.org>
Date:   2013-11-04 11:06:55 -0500

Use $ucc and $kcc as userspace and kernel compilers respectively

---

diff --git a/.farmrc b/.farmrc
index ba873a1..05d3f5e 100644
--- a/.farmrc
+++ b/.farmrc
@@ -5,4 +5,6 @@ v2.6.31 v2.6.32 v2.6.33 v2.6.34 v2.6.35 v2.6.36 v2.6.37 v2.6.38 v2.6.39
 v3.0 v3.1 v3.2 v3.3 v3.4 v3.5 v3.6 v3.7 v3.8 v3.9 v3.10 v3.11 v3.12
 linux-next/master"
 make="make -j11"
+kcc=gcc-4.4
+ucc=gcc
 export EXTRA_CFLAGS=-Werror-implicit-function-declaration
diff --git a/farm-compile b/farm-compile
index f506c6f..352f00c 100755
--- a/farm-compile
+++ b/farm-compile
@@ -28,7 +28,7 @@ for bintree in $farm/*; do
 	rm -f $log $log-ok $log-error
 	$make clean KERNELRELEASE=0 >/dev/null
 	echo -n "Checking $bintree "
-	if $make HOSTCC="gcc -DPATH_MAX=4096" CC="gcc-4.4 $ccflags" \
+	if $make HOSTCC="$ucc -DPATH_MAX=4096" CC="$kcc $ccflags" \
 	     ARCH=$arch KBUILD=$bintree KERNELPATH=$bintree C=2 \
 	     CF="-D__CHECK_ENDIAN__" $target >$log 2>&1; then
 		mv $log $log-ok
diff --git a/farm-config32 b/farm-config32
index 6f6e4bd..25ee7e4 100755
--- a/farm-config32
+++ b/farm-config32
@@ -13,7 +13,7 @@ for srctree in $farm/*; do
 	rm -rf $bintree-bad
 	mkdir $bintree
 	cd $srctree
-	$make HOSTCC="gcc -DPATH_MAX=4096" KBUILD_OUTPUT=$bintree \
+	$make HOSTCC="$ucc -DPATH_MAX=4096" KBUILD_OUTPUT=$bintree \
 		CONFIG_WIRELESS_EXT=y CONFIG_NET_RADIO=y CONFIG_IPW2100=y \
 		CONFIG_CRYPTO=y ARCH=$arch defconfig
 	cd $bintree
@@ -21,8 +21,8 @@ for srctree in $farm/*; do
 	sed -i -e 's/# CONFIG_WIRELESS_EXT is not set/CONFIG_WIRELESS_EXT=y/' .config
 	sed -i -e 's/# CONFIG_IPW2100 is not set/CONFIG_IPW2100=y/' .config
 	sed -i -e 's/# CONFIG_CRYPTO is not set/CONFIG_CRYPTO=y/' .config
-	yes "" | $make HOSTCC="gcc -DPATH_MAX=4096" ARCH=$arch oldconfig
-	if $make CC="gcc-4.4 $ccflags" HOSTCC="gcc -DPATH_MAX=4096" \
+	yes "" | $make HOSTCC="$ucc -DPATH_MAX=4096" ARCH=$arch oldconfig
+	if $make CC="$kcc $ccflags" HOSTCC="$ucc -DPATH_MAX=4096" \
 	     ARCH=$arch modules_prepare >Log-prepare 2>&1; then
 		date > $bintree/configured-$arch
 	else
diff --git a/farm-config64 b/farm-config64
index d30dd44..1413c5e 100755
--- a/farm-config64
+++ b/farm-config64
@@ -13,7 +13,7 @@ for srctree in $farm/*; do
 	rm -rf $bintree-bad
 	mkdir $bintree
 	cd $srctree
-	$make HOSTCC="gcc -DPATH_MAX=4096" KBUILD_OUTPUT=$bintree \
+	$make HOSTCC="$ucc -DPATH_MAX=4096" KBUILD_OUTPUT=$bintree \
 		CONFIG_WIRELESS_EXT=y CONFIG_NET_RADIO=y CONFIG_IPW2100=y \
 		CONFIG_CRYPTO=y ARCH=$arch defconfig
 	cd $bintree
@@ -21,8 +21,8 @@ for srctree in $farm/*; do
 	sed -i -e 's/# CONFIG_WIRELESS_EXT is not set/CONFIG_WIRELESS_EXT=y/' .config
 	sed -i -e 's/# CONFIG_IPW2100 is not set/CONFIG_IPW2100=y/' .config
 	sed -i -e 's/# CONFIG_CRYPTO is not set/CONFIG_CRYPTO=y/' .config
-	yes "" | $make HOSTCC="gcc -DPATH_MAX=4096" ARCH=$arch oldconfig
-	if $make CC="gcc-4.4 $ccflags" HOSTCC="gcc -DPATH_MAX=4096" \
+	yes "" | $make HOSTCC="$ucc -DPATH_MAX=4096" ARCH=$arch oldconfig
+	if $make CC="$kcc $ccflags" HOSTCC="$ucc -DPATH_MAX=4096" \
 	     ARCH=$arch modules_prepare >Log-prepare 2>&1; then
 		date > $bintree/configured-$arch
 	else
